operator/README.md

```diff
@@ -0,0 +1,275 @@
+# ADX Operator Guide
+
+> Scope: this document focuses exclusively on what the adx-mon operator does for Azure Data Explorer (ADX). Ingestor and Collector component operations are still a work-in-progress.
+
+## What the operator provides (ADX only)
```

Comment:
I think we should have prose describing the problem we solve with federation and a mermaid diagram with example deployments (one or two hubs, with several downstream partition clusters). Some description as well about the data residing in the partitions, but "one pane of glass" to query through the hubs. What problems does the operator solve?